lib/frontend/components/shared/gradient_icon.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:project_fomo/style.dart';
+
+
+// Definitely needs some work on... doesn't really fit the gradient perfect
+// Perfect task for Jack our super detailed dude
+// Linear Gradient will NOT work... it'll only fill with one color
+// this method of a gradient icon needs a radial gradient
+
+class GradientIcon extends StatelessWidget {
+
+  final IconData icon;
+  final double size;
+
+  GradientIcon({@required this.icon, @required this.size});
+
+  @override
+  Widget build(BuildContext context) {
+    return ShaderMask(
```

Comment:
Can the existing gradient in the stylesheet not be used here?

Reply:
It doesn't have the same effect. Unfortunately, that type of Gradient is just pure purple